proto/src/oauth2.rs

```diff
@@ -362,7 +362,8 @@ pub struct AccessTokenIntrospectResponse {
     pub sub: Option<String>,
     pub aud: Option<String>,
     pub iss: Option<String>,
-    pub jti: Option<String>,
+    // JWT ID <https://www.rfc-editor.org/rfc/rfc7519#section-4.1.7> set to session ID
+    pub jti: Uuid,
```

Comment:
Should be Option<Uuid> so that in the Introspect response we can default it to None.

Reply:
as discussed elsewhere, OAuth2RFC9068Token implies RFC9068 which makes it mandatory